docs/analog/commands/uplink/DayMC.md

```diff
@@ -9,13 +9,13 @@ This command is generated by the sensor and sent within a specified period witho
 
 ### Format
 
-| Size       | Type                                                         | Field                                               |
-| ---------- | ------------------------------------------------------------ | --------------------------------------------------- |
-| `1`        | `uint8`                                                      | command id = `0x16`                                 |
-| `1`        | `uint8`                                                      | command size (dynamic, `4+`)                        |
-| `2`        | [packed date](../../types.md#packed-date)                    | [date](#date)                                       |
-| `1..5`     | [extended value](../../types.md#extended-value)              | [channels bit set](../../types.md#channels-bit-set) |
-| `[1..5]*n` | sequence of [extended values](../../types.md#extended-value) | [channel values sequence](#channel-values)          |
+| Size       | Type                                                         | Field                                                         |
+| ---------- | ------------------------------------------------------------ | ------------------------------------------------------------- |
+| `1`        | `uint8`                                                      | command id = `0x16`                                           |
+| `1`        | `uint8`                                                      | command size (dynamic, `4+`)                                  |
+| `2`        | [packed date](../../types.md#packed-date)                    | [date](#date)                                                 |
+| `1..5`     | [extended value](../../types.md#extended-value)              | [channels bit set](../../parameter-types.md#channels-bit-set) |
```

Comment:
why change to parameter-types.md?

and in all other places

Reply:
I made a mistake, I put it back the way it was